### PR TITLE
Update going-to-production.asciidoc

### PR DESCRIPTION
**Pull request by kaviwe**
@Peppe

### CODE REVIEW

File: articles/ce/going-to-production.asciidoc [hunk 1]
```diff
@@ -8,33 +8,49 @@ layout: page
 = Setting Up for Production
 
 ifdef::articles[]
-Enabling the production mode is described in <<{articles}/guide/production/production-overview#, Deploying to Production>>.
+Enabling production mode for a Vaadin app is described in <<{articles}/guide/production/production-overview#, Deploying to Production>>.
```
Comment (Peppe) [inline articles/ce/going-to-production.asciidoc:11]:
```suggestion
Enabling production mode for a Vaadin application is described in <<{articles}/guide/production/production-overview#, Deploying to Production>>.
```

Not better, but consistent with Vaadin's documentation in general

File: articles/ce/going-to-production.asciidoc [hunk 2]
```diff
@@ -8,33 +8,49 @@ layout: page
...
 endif::articles[]
 ifndef::articles[]
-Enabling the production mode is described in <<../flow/production/tutorial-production-mode-basic#, Deploying to Production>>.
+Enabling production mode for a Vaadin app is described in <<../flow/production/tutorial-production-mode-basic#, Deploying to Production>>.
```
Comment (Peppe) [inline articles/ce/going-to-production.asciidoc:14]:
```suggestion
Enabling production mode for a Vaadin application is described in <<../flow/production/tutorial-production-mode-basic#, Deploying to Production>>.
```

File: articles/ce/going-to-production.asciidoc [hunk 3]
```diff
@@ -8,33 +8,49 @@ layout: page
...
 endif::articles[]
-Using Collaboration Engine requires some extra steps when you take your application to production.
+If your app uses Collaboration Engine, it requires some extra steps when you take your application to production. These steps are described here.
```
Comment (Peppe) [inline articles/ce/going-to-production.asciidoc:16]:
```suggestion
If your application uses Collaboration Engine, it requires some extra steps when you take your application to production.
These steps are described here.
```
Prefer application over app.
One sentence per line.

File: articles/ce/going-to-production.asciidoc [hunk 4]
```diff
@@ -8,33 +8,49 @@ layout: page
...
 
-You need to:
+Prior to configuring your Vaadin app for production mode, you need to:
```
Comment (Peppe) [inline articles/ce/going-to-production.asciidoc:18]:
```suggestion
Before configuring your Vaadin application for production mode, you need to:
```

File: articles/ce/going-to-production.asciidoc [hunk 5]
```diff
@@ -8,33 +8,49 @@ layout: page
...
 
-[[ce.production.obtain-license]]
-== Obtaining a License File
+[[ce.production.obtain-universal-license]]
+== Obtaining a Free "Universal" License File
 
-First, you need to have a subscription with a large enough user limit to meet your needs.
-If you do not yet have a license, https://vaadin.com/collaboration/#contact-us[request a license through the product page] to get in contact with Vaadin.
+All Vaadin apps receive a free 10 end-user per month quota with a universal license file. 
+If you don't have a commercial Vaadin subscription (Pro, Prime or Enterprise), https://website.vaadin.com/hubfs/collaboration-engine/ce-license.json[click here] to download the universal license file.
```
Comment (Peppe) [inline articles/ce/going-to-production.asciidoc:30]:
```suggestion
If you don't have a commercial Vaadin subscription (Pro, Prime, or Enterprise), https://website.vaadin.com/hubfs/collaboration-engine/ce-license.json[click here] to download the universal license file.
```

File: articles/ce/going-to-production.asciidoc [hunk 6]
```diff
@@ -8,33 +8,49 @@ layout: page
...
 
 The name of the license file is `ce-license.json`.
+It informs Collaboration Engine that you are entitled to receive a quota of 10 end users per month.
+
+The universal license file cannot be upgraded to increase the quota of monthly end users.
```
Comment (Peppe) [inline articles/ce/going-to-production.asciidoc:35]:
```suggestion
You can upgrade to a commercial license to increase your monthly end user quota.
```

I think you do upgrade the license file, when you purchase a bigger one. It is the same process as going from a small commercial quota to a bigger commercial quota.

File: articles/ce/going-to-production.asciidoc [hunk 7]
```diff
@@ -8,33 +8,49 @@ layout: page
...
+All Vaadin apps receive a free 10 end-user per month quota with a universal license file. 
+If you don't have a commercial Vaadin subscription (Pro, Prime or Enterprise), https://website.vaadin.com/hubfs/collaboration-engine/ce-license.json[click here] to download the universal license file.
 
 The name of the license file is `ce-license.json`.
+It informs Collaboration Engine that you are entitled to receive a quota of 10 end users per month.
+
+The universal license file cannot be upgraded to increase the quota of monthly end users. 
+
+[[ce.production.obtain-license]]
+== Obtaining a commercial license file
```
Comment (Peppe) [inline articles/ce/going-to-production.asciidoc:38]:
```suggestion
== Obtaining a Commercial License File
```
All titles in Docs should be title case.

File: articles/ce/going-to-production.asciidoc [hunk 8]
```diff
@@ -8,33 +8,49 @@ layout: page
...
+
+If you need to use Collaboration Engine powered features for more than 10 end users / month, please request a commercial license file.
+
+First, you need to have a commercial Vaadin subscription (Pro, Prime or Enterprise) in order to request this license. Each Vaadin subscription offers a bundled quota of end users per month, based on the number of development seats. Visit, https://vaadin.com/collaboration#pricing[the official feature page] to learn more about the bundled quotas.
```
Comment (Peppe) [inline articles/ce/going-to-production.asciidoc:42]:
```suggestion
First, you need to have a commercial Vaadin subscription (Pro, Prime, or Enterprise) to request this license.
Each Vaadin subscription offers a bundled quota of end users per month, based on the number of development seats.
Visit, https://vaadin.com/collaboration#pricing[the official feature page] to learn more about the bundled quotas. 
```

File: articles/ce/going-to-production.asciidoc [hunk 9]
```diff
@@ -8,33 +8,49 @@ layout: page
...
+The universal license file cannot be upgraded to increase the quota of monthly end users. 
+
+[[ce.production.obtain-license]]
+== Obtaining a commercial license file
+
+If you need to use Collaboration Engine powered features for more than 10 end users / month, please request a commercial license file.
+
+First, you need to have a commercial Vaadin subscription (Pro, Prime or Enterprise) in order to request this license. Each Vaadin subscription offers a bundled quota of end users per month, based on the number of development seats. Visit, https://vaadin.com/collaboration#pricing[the official feature page] to learn more about the bundled quotas. 
+
+To obtain your license, https://vaadin.com/collaboration/#contact-us[click here to contact our team]. You can also request and purchase a larger quota of end users than provided by your commercial Vaadin subscription. Additional end users above your bundled quota are billed at 1 USD (or 1 EUR) per month.
```
Comment (Peppe) [inline articles/ce/going-to-production.asciidoc:44]:
```suggestion
To obtain your license, https://vaadin.com/collaboration/#contact-us[click here to contact our team]. 
You can also request and purchase a larger quota of end users than provided by your commercial Vaadin subscription. 
Additional end users above your bundled quota are billed at 1 USD (or 1 EUR) per month.
```

File: articles/ce/going-to-production.asciidoc [hunk 10]
```diff
@@ -8,33 +8,49 @@ layout: page
...
+
+The name of the license file will be `ce-license.json`.
```
Comment (Peppe) [inline articles/ce/going-to-production.asciidoc:46]:
```suggestion
The name of the license file is `ce-license.json`.
```

**Review by Peppe — CHANGES_REQUESTED** [review]

File: articles/ce/going-to-production.asciidoc (hunk 6 above)
Comment (kaviwe) [reply articles/ce/going-to-production.asciidoc:36]: Ah, I see your point. That's not what I meant to say actually; maybe this is more reflective of my intent:

"You can upgrade to a commercial license to increase your monthly end user quota. However, a commercial license also requires an active Vaadin commercial subscription (Pro, Prime, or Enterprise)."

Not sure how to add a suggestion to your suggestion @Peppe

Comment (Peppe) [reply articles/ce/going-to-production.asciidoc:36]: Sounds good I changed the suggestion to match your text.

**Review by Peppe — APPROVED** [review]

**Comment by Peppe** [comment]
This patch should be gotten into V14 and latest. Do I make a copy of the PR into latest, or can it be somehow cherry-picked?

File: articles/ce/going-to-production.asciidoc [hunk 11]
```diff
@@ -8,33 +8,55 @@ layout: page
...
 . Configure where Collaboration Engine files are stored.
 . Store the license file in the right location.
-. Implement a `LicenseEventHandler` to be notified when the license needs to be updated.
+. Implement a `LicenseEventHandler` to get notified when the license needs to be updated.
 
 
+[[ce.production.obtain-universal-license]]
+== Obtaining a Free "Universal" License File
+
+All Vaadin apps receive a free 10 end-user per month quota with a universal license file.
```
Comment (jouni) [inline articles/ce/going-to-production.asciidoc:30]:
```suggestion
All Vaadin applications receive a free 10 end user per month quota with a universal license file. 
```
“end user” was used in other parts of the article. Please double check that you use it consistently.

File: articles/ce/going-to-production.asciidoc [hunk 12]
```diff
@@ -8,33 +8,55 @@ layout: page
...
 . Store the license file in the right location.
-. Implement a `LicenseEventHandler` to be notified when the license needs to be updated.
+. Implement a `LicenseEventHandler` to get notified when the license needs to be updated.
 
 
+[[ce.production.obtain-universal-license]]
+== Obtaining a Free "Universal" License File
+
+All Vaadin apps receive a free 10 end-user per month quota with a universal license file. 
+If you don't have a commercial Vaadin subscription (Pro, Prime, or Enterprise), https://website.vaadin.com/hubfs/collaboration-engine/ce-license.json[click here] to download the universal license file.
```
Comment (jouni) [inline articles/ce/going-to-production.asciidoc:31]:
```suggestion
If you don't have a commercial Vaadin subscription (Pro, Prime, or Enterprise), you can https://website.vaadin.com/hubfs/collaboration-engine/ce-license.json[download the universal license file].
```
Avoid “click here” links, those are bad for accessibility.

File: articles/ce/going-to-production.asciidoc [hunk 13]
```diff
@@ -8,33 +8,55 @@ layout: page
...
+Enabling production mode for a Vaadin application is described in <<../flow/production/tutorial-production-mode-basic#, Deploying to Production>>.
 endif::articles[]
-Using Collaboration Engine requires some extra steps when you take your application to production.
+If your application uses Collaboration Engine, it requires some extra steps when you take your application to production.
+These steps are described here.
 
-You need to:
+Before configuring your Vaadin application for production mode, you need to:
 
 . Get a license file with an end-user quota.
```
Comment (jouni) [inline articles/ce/going-to-production.asciidoc:21]:
```suggestion
. Get a license file with an end user quota.
```

File: articles/ce/going-to-production.asciidoc [hunk 14]
```diff
@@ -8,33 +8,55 @@ layout: page
...
+
+If you need to use Collaboration Engine powered features for more than 10 end users / month, please request a commercial license file.
 
-First, you need to have a subscription with a large enough user limit to meet your needs.
-If you do not yet have a license, https://vaadin.com/collaboration/#contact-us[request a license through the product page] to get in contact with Vaadin.
+First, you need to have a commercial Vaadin subscription (Pro, Prime, or Enterprise) to request this license.
+Each Vaadin subscription offers a bundled quota of end users per month, based on the number of development seats.
+Visit, https://vaadin.com/collaboration#pricing[the official feature page] to learn more about the bundled quotas. 
+
+To obtain your license, https://vaadin.com/collaboration/#contact-us[click here to contact our team].
```
Comment (jouni) [inline articles/ce/going-to-production.asciidoc:48]:
```suggestion
To obtain your license, https://vaadin.com/collaboration/#contact-us[contact our team]. 
```

File: articles/ce/going-to-production.asciidoc [hunk 15]
```diff
@@ -8,33 +8,55 @@ layout: page
...
 
-First, you need to have a subscription with a large enough user limit to meet your needs.
-If you do not yet have a license, https://vaadin.com/collaboration/#contact-us[request a license through the product page] to get in contact with Vaadin.
+First, you need to have a commercial Vaadin subscription (Pro, Prime, or Enterprise) to request this license.
+Each Vaadin subscription offers a bundled quota of end users per month, based on the number of development seats.
+Visit, https://vaadin.com/collaboration#pricing[the official feature page] to learn more about the bundled quotas. 
+
+To obtain your license, https://vaadin.com/collaboration/#contact-us[click here to contact our team]. 
+You can also request and purchase a larger quota of end users than provided by your commercial Vaadin subscription. 
+Additional end users above your bundled quota are billed at 1 USD (or 1 EUR) per month.
```
Comment (jouni) [inline articles/ce/going-to-production.asciidoc:50]:
```suggestion
End users above your bundled quota are billed at 1 USD (or 1 EUR) per month.
```

**Review by jouni — CHANGES_REQUESTED** [review]

**Review by jouni — APPROVED** [review]

**Pull request merged**